trackbasket_be/models/at_risk_user.py

```diff
@@ -11,13 +14,14 @@ class AtRiskUser(BaseModel, db.Model):
   state = db.Column(db.String)
   zipcode = db.Column(db.String)
   phone_number = db.Column(db.String)
+  store = db.relationship('Store', backref=backref('at_risk_user', uselist=False))
```

Comment:
creates the one-to-one relationship between at `at_risk_user` and the closest `store` (`uselist` flag would be set to true for a one-to-many relationship)